README.md

```diff
@@ -1 +1,2 @@
-# test-your-lexicon
\ No newline at end of file
+# How to run
```

Comment:
Let's expand this readme. Currently, it does not have much information and it can be tricky to understand what is stored in this repo and how one should run it.

Usually good Readme file has:
- Short project description
- Short technical description (what tech stack is used)
- Prerequisites that are needed to build / run the app. Your project has at least one - Docker - to be installed
- Useful commands. At this point we'll need to add: how to run the app, how to run migrations and how to run checkstyle.

Reply:
I have flyway as Spring dependency, should I remove it as dependency and at it as plugin so we can run it from command line?

Reply:
Yes, let's do that